src/BuiltInTools/dotnet-watch/CommandLineOptions.cs

```diff
@@ -107,8 +106,13 @@ internal sealed class CommandLineOptions
         rootCommand.Options.Add(shortProjectOption);
         rootCommand.Options.Add(launchProfileOption);
         rootCommand.Options.Add(noLaunchProfileOption);
-        rootCommand.Options.Add(targetFrameworkOption);
-        rootCommand.Options.Add(propertyOption);
+
+        var buildOptions = RunCommandParser.GetCommand().Options.Where(o => o is IForwardedOption);
```

Comment:
Just realized - different subcommands have different forwarded options, so we'll need to use specific options to each subcommand...